README.md

```diff
@@ -20,35 +20,52 @@ Distribution Alternatives:
 
 ### Installation
 
-* Backup your previous configuration (if any exists)
-
-### Archive Installation
-* On the home/landing page for the project find the blue "<> CODE" button click it and select Local > Download ZIP.
-* Extract the archive to:
-  `~/.config/nvim` (Linux)
-  `~/.config/nvim` (MacOS)
-  `%userprofile%\AppData\Local\nvim\` (Windows)
-* Ensure your extraction method did not extract with a parent folder. For example in ~/.config/nvim you should have init.lua not another folder called kickstart.nvim.
-
-### Git Clone Installation
-* From a terminal cd/dir to:
-    `~/.config/nvim` (Linux)
-    `~/.config/nvim` (MacOS)
-    `%userprofile%\AppData\Local\nvim\` (Windows)
-
-* Run: `git clone https://github.com/nvim-lua/kickstart.nvim.git ~/.config/nvim` OR: `gh repo clone nvim-lua/kickstart.nvim`
-* Run Neovim (from terminal or shortcut) and allow lazy.nvim to download files and set up the basics.
-* Once the setup is complete, restart Neovim.
-* **You're ready to go!**
-
-* (Recommended/Optional) Fork this repo (so that you have your own copy that you can modify).
-* Clone the kickstart repo into `$HOME/.config/nvim/` (Linux/Mac) or `%userprofile%\AppData\Local\nvim\` (Windows)
-  * If you don't want to include it as a git repo, you can just clone it and then move the files to this location
-
-Additional system requirements:
-- Make sure to review the readmes of the plugins if you are experiencing errors. In particular:
-  - [ripgrep](https://github.com/BurntSushi/ripgrep#installation) is required for multiple [telescope](https://github.com/nvim-telescope/telescope.nvim#suggested-dependencies) pickers.
-- See [Windows Installation](#Windows-Installation) if you have trouble with `telescope-fzf-native`
+> **NOTE**  
+> [Backup](#FAQ) your previous configuration (if any exists)
+
+Requirements:
+* Make sure to review the readmes of the plugins if you are experiencing errors. In particular:
+  * [ripgrep](https://github.com/BurntSushi/ripgrep#installation) is required for multiple [telescope](https://github.com/nvim-telescope/telescope.nvim#suggested-dependencies) pickers.
+* See [Windows Installation](#Windows-Installation) if you have trouble with `telescope-fzf-native`
+
+Neovim's configurations are located under the following paths, depending on your OS:
+
+| OS | PATH |
+| :- | :--- |
+| Linux | `~/.config/nvim` |
+| MacOS | `~/.config/nvim` |
+| Windows | `%userprofile%\AppData\Local\nvim\` |
+
+#### Archive Installation
+
+* On the home/landing page for the project find the green "`<> CODE`" button click it and select `Local > Download ZIP`.
```

Comment:
Hi. I can appreciate that you're trying to simplify here but I disagree with this change and feel it represents a negative departure from the current instructions.

If people use Git to clone the repo locally, they can then use `git pull` to update the repo from upstream as changes happen.

With your method, you've stripped that capability away which I see as a dis-service to new users.

Reply:
FWIW I've just used what was already on the README. We are actually on the same page here, I will much prefer that git was the preferred, and frankly only, option for installing kickstart

Reply:
OK thank you that makes sense. Wanna fix it and replace the unzip instructions with git clone instructions? :)